src/main/java/edu/ucsb/cs156/gauchoride/config/SecurityConfig.java

```diff
@@ -88,6 +88,13 @@ private GrantedAuthoritiesMapper userAuthoritiesMapper() {
           if (email.endsWith("@ucsb.edu")) {
             mappedAuthorities.add(new SimpleGrantedAuthority("ROLE_MEMBER"));
           }
+          
+          Optional<User> u = userRepository.findByEmail(email);
+          if (u.isPresent() && u.get().getDriver()) {
+            mappedAuthorities.add(new SimpleGrantedAuthority("ROLE_DRIVER"));
+          } else {
+            mappedAuthorities.add(new SimpleGrantedAuthority("ROLE_RIDER"));
+          }
```

Comment:
Doesn't belong in this PR... 

The database and repository don't depend on it, and that code is already in a separate PR.

Try to develop the habits to keep separate PRs separate.

Reply:
Ok. If I'm working on CRUD requests, and was awaiting approval of the PR which was adding driver and rider roles, in that case would it be fine to branch off of the branch awaiting review since they depend on each other